tests/php/Centreon/Infrastructure/RequestParameters/SqlRequestParametersTranslator.php

```diff
@@ -0,0 +1,194 @@
+<?php
```

Comment:
the file name has to has suffix `Test` as SqlRequestParametersTranslatorTest.php